Configure

```diff
@@ -1509,9 +1509,7 @@ unless ($disabled{asan} || defined $detected_sanitizers{asan}) {
 }
 
 unless ($disabled{ubsan} || defined $detected_sanitizers{ubsan}) {
-    # -DPEDANTIC or -fnosanitize=alignment may also be required on some
-    # platforms.
-    push @{$config{cflags}}, "-fsanitize=undefined", "-fno-sanitize-recover=all";
+    push @{$config{cflags}}, "-fsanitize=undefined", "-fno-sanitize-recover=all", "-DPEDANTIC";
```

Comment:
`-DPREDANTIC` isn't needed on `linux-aarch64` - will defining it mean that some code paths that are currently checked won't be checked in the future?

Reply:
``` console
$ git grep -E '# *if.*PEDANTIC'
apps/passwd.c:# ifndef PEDANTIC              /* Unfortunately, this generates a "no
crypto/bn/bn_local.h:# if !defined(OPENSSL_NO_ASM) && !defined(OPENSSL_NO_INLINE_ASM) && !defined(PEDANTIC)
crypto/chacha/chacha_enc.c:# ifndef PEDANTIC
crypto/modes/cbc128.c:#if !defined(STRICT_ALIGNMENT) && !defined(PEDANTIC)
crypto/sha/sha256.c:# ifndef PEDANTIC
crypto/sha/sha512.c:# ifndef PEDANTIC
crypto/sm3/sm3_local.h:#ifndef PEDANTIC
crypto/whrlpool/wp_block.c:#if !defined(PEDANTIC) && (defined(__i386) || defined(__i386__) || \
crypto/whrlpool/wp_block.c:#ifndef PEDANTIC
engines/e_afalg.c:# ifndef PEDANTIC
include/crypto/md32_common.h:#ifndef PEDANTIC
include/crypto/modes.h:#ifndef PEDANTIC
include/crypto/modes.h:#if !defined(PEDANTIC) && !defined(OPENSSL_NO_ASM) && !defined(OPENSSL_NO_INLINE_ASM)
include/internal/ktls.h:#   ifndef PEDANTIC
include/internal/ktls.h:#    ifndef PEDANTIC
test/build_wincrypt_test.c:#  ifndef PEDANTIC
test/cmp_ctx_test.c:#ifndef PEDANTIC
```

Reply:
As I recall, the blocks that are excluded with `-DPEDANTIC` are small and not very significant...  that being said, I haven't analyzed this further than the grep above.

Reply:
hmm, it does look as if the main use is disabling some `asm` blocks and turning off `STRICT_ALIGNMENT`. so, okay, this doesn't look too bad :)